cowtown2023/src/main/java/com/team2357/frc2023/commands/intakeRoller/IntakeRollerEjectCubeCommand.java

```diff
@@ -0,0 +1,28 @@
+package com.team2357.frc2023.commands.intakeRoller;
+
+import com.team2357.frc2023.Robot;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class IntakeRollerEjectCubeCommand extends CommandBase {
+
```

Comment:
Not a big deal for these little commands, and now that we are not using singletons with long names, but keep in mind that you can set the intake subsystem to a local variable, that way you do not have to do Robot.intake everytime you want to reference it. Nothing needs to change here unless you want to.